src/vehicle.cpp

```diff
@@ -3756,9 +3766,49 @@ void vehicle::idle(bool on_map) {
 
     if( on_map ) {
         update_time();
+        if(scoop_on){
+            operate_scoop();
+        }
+    }
+}
+void vehicle::operate_scoop()
+{
+    std::vector<int> scoops = all_parts_with_feature( "SCOOP" );
+    auto veh_points = get_points();
+    for( int scoop : scoops ) {
+        int chance_to_damage_item = parts[scoop].info().dmg_mod;
+        int max_pickup_size = parts[scoop].info().bonus;
+        const char *sound_msgs[] = {"Whirrrr", "Ker-chunk", "Swish", "Cugugugugug"};
```

Comment:
They were supposed to be onomatopoeia. Now most languages have their own preferred way to spell and pronounce the same sounds. 

What do you suppose I do with them?